packages/paste-website/src/pages/patterns/error-state/index.mdx

```diff
@@ -0,0 +1,531 @@
+export const meta = {
+  title: 'Error state',
+  description: 'An error state notifies a user that something went wrong.',
+  slug: '/patterns/error-state/',
+};
+
+import {Anchor} from '@twilio-paste/anchor';
+import {Alert} from '@twilio-paste/alert';
+import {Box} from '@twilio-paste/box';
+import {Card} from '@twilio-paste/card';
+import {Label} from '@twilio-paste/label';
+import {Input} from '@twilio-paste/input';
+import {HelpText} from '@twilio-paste/help-text';
+import {Callout, CalloutHeading, CalloutText} from '@twilio-paste/callout';
+import {Toast} from '@twilio-paste/toast';
+import {Table, THead, Tr, Th, TBody, Td, TFoot} from '@twilio-paste/table';
+import {DoDont, Do, Dont} from '../../../components/DoDont';
+import DefaultLayout from '../../../layouts/DefaultLayout';
+import {getFeature, getNavigationData} from '../../../utils/api';
+
+export default DefaultLayout;
+
+export const getStaticProps = async () => {
+  const navigationData = await getNavigationData();
+  const feature = await getFeature('Error state');
+  return {
+    props: {
+      data: {
+        ...feature,
+      },
+      navigationData,
+    },
+  };
+};
+
+<content>
+
+<NormalizedPatternHeader description={meta.description} name={meta.title} data={props.data} />
+
+</content>
+
+---
+
+<contentwrapper>
+
+<PageAside data={mdxHeadings} />
+
+<content>
+
```

Comment:
todo: add an Ingredients section at the top to match other Patterns pages. Include Alert, Callout, Toast, Help Text.

Reply:
I left this section out because it's kind of different from the other patterns in that we're not really composing 1 _thing_ with many ingredients. It felt more like guidance towards one of many possible outcomes. Can still include it though, just wanted to give context

Reply:
ok added the section in and changed the comment saying to import all of the following components to import only one of the following components